app/models/spotlight/resources/upload.rb

```diff
@@ -4,7 +4,7 @@ module Resources
     ##
     # Exhibit-specific resources, created using uploaded and custom fields
     class Upload < Spotlight::Resource
-      belongs_to :upload, class_name: 'Spotlight::FeaturedImage', optional: true
+      belongs_to :upload, class_name: 'Spotlight::FeaturedImage', optional: true, validate: true
```

Comment:
You might want to drop a comment in here about why we need to validate the upload.